### DISCUSSION TITLE
No dashboard on fresh Snap install of Semaphore

### DISCUSSION THREAD

**Discussion by no-good-usernames-left** [semaphoreui/semaphore#924] [General]
Brand new Ubuntu 22 

Installed Semaphore via Snap and all the thing does is prompt to make a new project, there's no dashboard, no inventory, no environment, no team, nothing! 

What causes this and how to fix?

**Comment by fiftin** [comment]
Project contains inventory, env, etc. You should project firstly.

**Reply by no-good-usernames-left** [reply]
I did and made multiple projects, no improvement or visibility to the rest of the Semaphore instance. 

Scrapped 22 and went to 20 and it works as designed now

**Comment by tboerger** [comment]
Closing this as it can't be reproduced. If this issue still exists for you please create a new discussion and provide as much details as possible.
